pybamm/callbacks.py

```diff
@@ -201,20 +201,6 @@ def on_cycle_end(self, logs):
                     f"is below stopping capacity ({cap_stop:.3f} Ah)."
                 )
 
-        voltage_stop = logs["stopping conditions"]["voltage"]
```

Comment:
We can't do this if we don't allow calculating summary variables on the fly. I'm not sure how useful setting a voltage cut-off at the experiment level in this way is anyway? Surely we would set cut-offs at the step level?

Reply:
It was added in response to this request https://github.com/pybamm-team/PyBaMM/issues/1826. I completely forgot we had this feature but it's quite useful for GITT! The lines commented here are just for logging, the line that does the check is in the `Simulation` class, we should keep it but evaluate the voltage inside the `Simulation` class instead of using the summary variable. It would only get called if `voltage_stop is not None`, so no performance hit in general.